test/new_relic/slack_notifications_tests/cve_notifier_tests.rb

```diff
@@ -0,0 +1,23 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'minitest/autorun'
+require_relative '../../../.github/workflows/scripts/slack_notifications/cve_notifier'
+
+class SlackNotifications < Minitest::Test
```

Comment:
Ooh! Tiny thing, could this be renamed to CveNotifierTests?

Reply:
Ahh!! Yes thank you

Reply:
In fact, all of those test classes could've been named better. Fixed: [CveNotifierTests](https://github.com/newrelic/newrelic-ruby-agent/pull/1432/files#diff-e23b8415de0f7014dc906b5f630621f8a757036a01de8591fb249d2b41047b78R9), [GemNotifierTests](https://github.com/newrelic/newrelic-ruby-agent/pull/1432/files#diff-41275d956216f3157b605d345f4b89a6ed831343f8c73d1b687d5632fbb9f2c7R9), [SlackNotifierTests](https://github.com/newrelic/newrelic-ruby-agent/pull/1432/files#diff-30d914ac7129feb2d13bf43e052cf29fe91564381ac3623355ac7f02ae77d096)

Reply:
🦖 🦖 🦖